src/handlers/document_save.rs

```diff
@@ -6,10 +6,12 @@ use crate::shared;
 #[derive(Default)]
 pub struct DocumentSaveHandler {}
 
+#[async_trait::async_trait]
```

Comment:
is this a typo? other files are using:

```
#[async_trait]
```

Reply:
I have use statements that import it in the other files, in the files where its a one off I just use the full namespace.